[2023-12-26T10:08:26.340+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.extract_and_filter_customer manual__2023-12-26T10:08:22.215301+00:00 [queued]>
[2023-12-26T10:08:26.349+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.extract_and_filter_customer manual__2023-12-26T10:08:22.215301+00:00 [queued]>
[2023-12-26T10:08:26.350+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-26T10:08:26.350+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-12-26T10:08:26.351+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-26T10:08:26.365+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): extract_and_filter_customer> on 2023-12-26 10:08:22.215301+00:00
[2023-12-26T10:08:26.373+0000] {standard_task_runner.py:55} INFO - Started process 166 to run task
[2023-12-26T10:08:26.376+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'potential_customers_process', 'extract_and_filter_customer', 'manual__2023-12-26T10:08:22.215301+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/potential_customers_process.py', '--cfg-path', '/tmp/tmph61wm63y']
[2023-12-26T10:08:26.378+0000] {standard_task_runner.py:83} INFO - Job 24: Subtask extract_and_filter_customer
[2023-12-26T10:08:26.421+0000] {task_command.py:376} INFO - Running <TaskInstance: potential_customers_process.extract_and_filter_customer manual__2023-12-26T10:08:22.215301+00:00 [running]> on host 7d99470aba0c
[2023-12-26T10:08:26.459+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=potential_customers_process
AIRFLOW_CTX_TASK_ID=extract_and_filter_customer
AIRFLOW_CTX_EXECUTION_DATE=2023-12-26T10:08:22.215301+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-26T10:08:22.215301+00:00
[2023-12-26T10:08:26.928+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/potential_customers_process.py", line 40, in _extract_and_filter_customer
    female_condition = data["Gender"].lower() == "female"
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 5487, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'Series' object has no attribute 'lower'
[2023-12-26T10:08:26.939+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=potential_customers_process, task_id=extract_and_filter_customer, execution_date=20231226T100822, start_date=20231226T100826, end_date=20231226T100826
[2023-12-26T10:08:26.947+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 24 for task extract_and_filter_customer ('Series' object has no attribute 'lower'; 166)
[2023-12-26T10:08:26.963+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-12-26T10:08:26.985+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
